apps/eth/rpc.py

```diff
@@ -13,7 +13,8 @@
 class EventRPCClient(RPCClient):
 
     # pylint:disable=too-many-arguments
-    def subscribe(self, project, version, url=Event.get_event_subscription_url(), interval=5000, events=None):
+    def subscribe(self, project, version, url=Event.get_event_subscription_url(), interval=5000, events=None,
+                  blockheight=0):
```

Comment:
Thinking through how this will work in a normal Transmission boot sequence, we do not want the default value to be `0` here.  That will cause every boot of Transmission to reset the `lastBlock` to 0 for the subscription and cause every event to be re-processed.

Instead, this should default to `None` if not provided to this method.  Then below, if this parameter is None, then it should not be provided to Engine.